package.json

```diff
@@ -32,20 +32,20 @@
         "test": "echo \"Error: no tests\""
     },
     "peerDependencies": {
-        "@kapeta/ui-web-types": "<2",
-        "@kapeta/ui-web-utils": "<2",
+        "@kapeta/ui-web-types": "^1.3.1",
+        "@kapeta/ui-web-utils": "^0.0.29",
```

Comment:
Not sure we want to narrow these?

Reply:
Ok, changed it to `<2` and `<1` respectively

Reply:
I'd say leave them as open as we can, while ensuring compatibility. i.e. `>=0.0.29 < 1` is fine, if `0.0.29` has something we need. But to optimize the dependency tree, open is good :)